documentation/style-guide.md

```diff
@@ -0,0 +1,179 @@
+# Docs Style Guide
+
+Welcome to Codecademy Docs! This document is a general guide about entry standards. Following this guide will reduce the number of potential edits that will need to be made to your PR.
+
+## Types of Entries
+
+There are three types of Docs entries:
+
+### Topics
+- Written by the maintainers as needed.
+- Cover a broad category such as a programming languages like [Java](https://www.codecademy.com/resources/docs/java) and [C++](https://www.codecademy.com/resources/docs/cpp), or extensive subjects like ([Open Source](https://www.codecademy.com/resources/docs/open-source) and [Cybersecurity](https://www.codecademy.com/resources/docs/cybersecurity).
+- Vary in structure from a couple of paragraphs to a multiple-sectioned entry.
+### Concepts
+- Provided as a list under each [topic](#topics).
+- Cover a small piece of the parent topic such as a class, library, control structure, or language element.
```

Comment:
Consider adding this link to class: https://www.codecademy.com/resources/docs/general/class